README.md

```diff
@@ -42,14 +42,21 @@ optional arguments:
 You can list the features supported by the tools by using `./pico_project --list`. These features can
 be added to the project using the `--feature` options, this can be used multiple times.
 
-
-
 ## GUI version
 
 The GUI version of the tool, run by adding `--gui` to the command line, uses `tkinter` to provide a platform agnostic script that will run on Linux, Mac and Windows. All the options from the command line tool are also supported in the GUI.
 
 You can add specific features to your project by selecting them from the check boxes on the GUI. This will ensure the build system adds the appropriate code to the build, and also adds simple example code to the project showing how to use the feature. There are a number of options available, which provide the following functionality.
 
+## Installation using Python Pip (Linux)
```

Comment:
This section is in the wrong place in the readme. Please rebase on the latest readme, then I think probably best to move this to the bottom of the page?

Reply:
Probably worth clarifying that this is optional too, as it isn't how we (currently) document usage of pico-project-generator.